CODEOWNERS

```diff
@@ -15,17 +16,21 @@
 # @cilium/endpoint           Endpoint package
 # @cilium/github-sec         GitHub security (handling of secrets, consequences of pull_request_target, etc.)
 # @cilium/health             Cilium cluster health tool
+# @cilium/helm               Helm charts and best practices
 # @cilium/hubble             Hubble integration
 # @cilium/ipam               IPAM
+# @cilium/ipcache            Maintenance of pkg/ipcache
 # @cilium/kubernetes         K8s integration, K8s CNI plugin
 # @cilium/kvstore            Key/Value store: Consul, etcd
 # @cilium/loadbalancer       Load balancer
 # @cilium/loader             All related to LLVM, bpftool, Cilium loader, templating, etc.
+# @cilium/metrics            Metrics of the Cilium Agent
 # @cilium/operator           Cilium operator
 # @cilium/policy             Policy behaviour
 # @cilium/proxy              L7 proxy, Envoy
 # @cilium/vendor             Vendoring, dependency management
 # @cilium/wireguard          Wireguard integration
+# @cilium/nonexistantteam    Fake team to tell GitHub to skip assignment
```

Comment:
Which packages would these be?

Reply:
```console
$ grep 'nonexistantteam' CODEOWNERS
/Documentation/cmdref/ @cilium/nonexistantteam
```
I didn't invent any new team here, I just added the teams that were used in the file but not documented at the top.

Reply:
That was added on purpose, to avoid assigning tophat as a reviewer just because some generated files were regenerated.